README/ReleaseNotes/v630/index.md

```diff
@@ -283,6 +283,68 @@ Some of these classes are now removed from the public interface:
 
 ## PROOF Libraries
 
+## PyROOT
+
+### Changed ownership policy for non-`const` pointer member function parameters
+
+If you have a member function taking a raw pointer, like `MyClass::foo(T
+*obj)`, PyROOT was so far assuming that calling this method on `my_instance`
+transfers the ownership of `obj` to `my_instance`.
+
+However, this resulted in many memory leaks, since many functions with such a
+signature actually don't take ownership of the object.
+
+To avoid such memory leaks, PyROOT now doesn't make this guess anymore as of
+ROOT 6.30. Because of this change, some double-deletes or dangling references
+might creep up in your scripts. These need to be fixedby properly managing
+object lifetime with Python references.
+
+You can fix the dangling references problem for example via:
+
+  1. Assigning the object to a python variable
+  2. Creating an owning collection that keeps the objects alive
+  3. Writing a pythonization for the member function that does the ownership
+     transfer if needed
+  
+The double-delete problems can be fixed via:
+
+  1. Drop the ownership on the Python side with `ROOT.SetOvnership(obj, False)`
+  3. Writing a pythonization for the member function that drops the ownership on the Python side as above
+
+If there is a problem that you think might also affect also other users and can
+be fixed by a pythonization inside ROOT, please don't hesitate to open a GitHub
+issue about it!
```

Comment:
I wouldn't write this out explicitly in release notes, users know/should know they can open an issue if they have a problem

Reply:
You are right :+1: